docs/src/app/components/pages/components/DropDownMenu/ExampleSimple.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import DropDownMenu from 'material-ui/lib/DropDownMenu';
+import MenuItem from 'material-ui/lib/menus/menu-item';
+
+export default class DropDownMenuSimpleExample extends React.Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {value: 2};
+  }
+
+  handleChange = (e, index, value) => this.setState({value});
+
+  render() {
+    return (
+      <DropDownMenu value={this.state.value} onChange={this.handleChange}>
```

Comment:
That's not what I meant :D :D :D :D 

I mean, we have a huge number of deprecated API for this. I can't implement this with what we have now. I think it's more productive to wait for 0.15.0 to remove all the deprecated API and then support uncontrolled component for this. The code to remove is so confusing as it is. 0.15.0 we'll trim the size in half :D :D  :grin: